prowler/providers/azure/services/iam/iam_custom_role_permits_administering_resource_locks/iam_custom_role_permits_administering_resource_locks.py

```diff
@@ -0,0 +1,31 @@
+from re import search
+
+from prowler.lib.check.models import Check, Check_Report_Azure
+from prowler.providers.azure.services.iam.iam_client import iam_client
+
+
+class iam_custom_role_permits_administering_resource_locks(Check):
+    def execute(self) -> Check_Report_Azure:
+        findings = []
+        for subscription, roles in iam_client.roles.items():
+            roles = [role for role in roles if role.type == "CustomRole"]
```

Comment:
What if we store the custom roles within the IAM class, then you can reutilize that in some checks, getting that with `iam.custom_roles`.

Reply:
Done!